server/clients/manageRecallsApi/manageRecallsApiClient.ts

```diff
@@ -1,7 +1,7 @@
 import superagent from 'superagent'
 import { manageRecallsApiConfig } from '../../config'
 import RestClient from '../../data/restClient'
-import { RecallResponse as Recall } from '../../@types/manage-recalls-api/models/RecallResponse'
+import { RecallResponse, RecallResponse as Recall } from '../../@types/manage-recalls-api/models/RecallResponse'
```

Comment:
why do we need to import RecallResponse twice?

Reply:
ooh, dunno. some magic IntelliJ stuff. will remove